### PR TITLE
Fix Feedback FAB margins

### LINKED ISSUE #859: Navigation UI: Feedback Button
`Device: Motorola TC55`

The Audio and Feedback Buttons Do Not Align Up - See Below.

screenshot_2018-04-12-17-16-19

### PR DESCRIPTION
**Pull request by Guardiola31337**
- Fixes #859 

- Fixes Feedback FAB margins (matching Sound FAB ones)

Noting that this only happened in devices with low resolution (e.g. Motorola TC55) so friendly reminder here that we should test on those devices as well.

feedbackbuttonproperlyalignedlowresdevice

In any case, the distance between sound and feedback buttons is not optimal for that resolution ☝️ (too long IMO), should we consider adding layout support for lower resolutions?

cc @devotaaabel @danesfeder

### CODE REVIEW

**Comment by danesfeder** [comment]
>  should we consider adding layout support for lower resolutions

Can we add a layout with lower res qualifier to address this.  Dropping the margins to `8dp` will result in the buttons being too close on high-res devices like the Pixel 2

File: libandroid-navigation-ui/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="fab_margin_bottom">8dp</dimen>
```
Comment (danesfeder) [inline libandroid-navigation-ui/src/main/res/values/dimens.xml]: Can we adjust this to `16dp`?  On higher res devices, the buttons are too close imo, may be more easy to click the wrong one:

device-2018-04-24-083032

Comment (Guardiola31337) [reply libandroid-navigation-ui/src/main/res/values/dimens.xml]: Nah, that's not necessary 👀 `fab_margin_bottom` which doesn't affect the distance between sound and feedback buttons (it's the same exact value as it was).

**Comment by Guardiola31337** [comment]
Added `dimens.xml` "default" and `mdpi`, `hdpi`, `xhdpi`, `xxhdpi` and `xxxhdpi` values support in https://github.com/mapbox/mapbox-navigation-android/pull/878/commits/513476368d5d6d8d605fff9d3c0cf76d28172444

> BTW we should revisit all hardcoded "dimens" values and 👀 if we want to support any other configurations (screen densities, sizes, etc.).

And also adjust `hdpi`, `xhdpi`, `xxhdpi` and `xxxhdpi` values so look ✨ for each screen density (now all of them define the same values as the "default" ones).

@danesfeder could you confirm that https://github.com/mapbox/mapbox-navigation-android/pull/878#discussion_r183711397 is working as expected now?

**Comment by Danny-James** [comment]
@danesfeder should this now be available in the `13.0-SNAPSHOT` ?

**Comment by danesfeder** [comment]
@Danny-James yes 👍

**Comment by Danny-James** [comment]
@danesfeder Great Stuff, I've tested this and it's working.

**Comment by Guardiola31337** [comment]
@Danny-James Great to hear, thanks for the update.

**Pull request merged**